### ISSUE TITLE
Pop!_Shop updates Firefox when it's not installed

### ISSUE THREAD

**Issue by KloudKoder** [pop-os/pop#478]
**Distribution (run `cat /etc/os-release`):**
 
 All
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

Pop!_Shop

**Issue/Bug Description:**

I don't have Firefox. But for some reason my latest updated just installed Firefox components.

**Steps to reproduce (if you know):**

Install Pop OS. Uninstall Firefox. Reboot just to make sure everything has settled down. Apply all updates. Notice that Firefox components get updated as part of the main system updates.

**Expected behavior:**

Don't install unnecessary components. (I realize this isn't possible in the extreme sense, but the less, the better.)

**Other Notes:**

You might look at Solus' eopkg, which is a better designed package manager than apt. (Granted, their own software store is also bug-ridden like with any other Linux.)

**Comment by mmstick** [comment]
This is likely more of an issue with PackageKit cache than apt.

**Comment by KloudKoder** [comment]
Fair enough. Is there any way to fix this behavior, or does it make sense to switch to eopkg or some other alternative? Most likely this isn't going to be the only such issue.
